xarray/backends/zarr.py

```diff
@@ -563,6 +568,8 @@ def open_zarr(
     consolidated : bool, optional
         Whether to open the store using zarr's consolidated metadata
         capability. Only works for stores that have already been consolidated.
+    chunk_store : MutableMapping, optional
+        A separate Zarr store only for chunk data.
```

Comment:
Since we didn't explicitly handle the version requirement discussed [here](https://github.com/pydata/xarray/pull/3804#discussion_r445156318). I think we should indicate that this kwarg has a minimum zarr version requirement in the docstring.

Reply:
Done, in de8ee28e.